Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/entity/ContactDiaryLocationEntity.kt

```diff
@@ -9,7 +9,10 @@ import de.rki.coronawarnapp.contactdiary.model.ContactDiaryLocation
 data class ContactDiaryLocationEntity(
     @PrimaryKey(autoGenerate = true) @ColumnInfo(name = "locationId") override val locationId: Long = 0L,
     @ColumnInfo(name = "locationName") override var locationName: String
-) : ContactDiaryLocation
+) : ContactDiaryLocation {
```

Comment:
We could keep database entities clean by defining an extension in the UI layer like val ContactDiaryLocationEntity.stableId = locationId